tests/gui/pages/main.spec.js

```diff
@@ -147,6 +150,33 @@ describe('Browser: MainPage', function() {
       m.chai.expect(_.sortBy(extensions)).to.deep.equal(_.sortBy(SupportedFormatsModel.getAllExtensions()));
     });
 
+    describe('.getImageBasename()', function() {
+
+      it('should return the basename of the selected image', function() {
+        const controller = $controller('ImageSelectionController', {
+          $scope: {}
+        });
+
+        SelectionStateModel.setImage({
+          path: path.join(__dirname, 'foo', 'bar.img'),
+          size: 999999999
+        });
```

Comment:
Seems like lots of different unit tests now all end up creating their own temporary `image` objects. Would it be worth having a standard / shared test-image-object that all the unit-tests could re-use, or at least use as a common starting point? Or is it better for the unit tests to remain verbose and "independent" of each other?